src/main/java/frc/robot/subsystems/tiltedelevator/TiltedElevatorSubsystem.java

```diff
@@ -269,7 +269,10 @@ public ElevatorState getState() {
      * @param power The power to change the offset by.
      */
     public void changeOffsetDistMeters(double power) {
-        this.offsetDistMeters += OFFSET_FACTOR * power;
+        // if (we are below the top OR we are going down) AND (we are above the bottom OR going up), let the driver offset
+        if((getExtensionMeters() < EXTENSION_LIMIT_METERS || power < 0) && (getExtensionMeters() > 0 || power > 0)){
+            this.offsetDistMeters += OFFSET_FACTOR * power;
```

Comment:
@penguin212 what if we did a
```java
this.offsetDistMeters = MathUtil.clamp(
    offsetDistMeters = OFFSET_FACTOR,
    // distance from current extension to 0
    // distance from current extension to EXTENSION_LIMIT
);
```

Reply:
I realized I did a typo when writing out this code block in GitHub, @penguin212 can you update the `offsetDistMeters =` to a `offsetDistMeters +`? It doesn't make sense to set the offset to the offset factor 😔